src/pudl/output/pudltabl.py

```diff
@@ -162,9 +172,166 @@ def pu_ferc1(self, update=False):
                 self.pudl_engine)
         return self._dfs['pu_ferc1']
 
+    ###########################################################################
+    # EIA 861 Interim Outputs (awaiting full DB integration)
+    ###########################################################################
+    def etl_eia861(self, update=False):
+        """
+        A single function that runs the temporary EIA 861 ETL and sets all DFs.
+
+        This is an interim solution that provides a (somewhat) standard way of accessing
+        the EIA 861 data prior to its being fully integrated into the PUDL database. If
+        any of the dataframes is attempted to be accessed, all of them are set. Only
+        the tables that have actual transform functions are included, and as new
+        transform functions are completed, they would need to be added to the list
+        below. Surely there is a way to do this automatically / magically but that's
+        beyond my knowledge right now.
+
+        Args:
+            update (bool): Whether to overwrite the existing dataframes if they exist.
+
+        """
+        if update or self._dfs["balancing_authority_eia861"] is None:
```

Comment:
I thought about doing this, but in practice almost all of the work of generating the EIA 861 data is reading the spreadsheets, and doing the whole thing takes less than 1 minute.

Because *all* of the tables are generated whenever *any* of them are, just checking to see if one of them is present is enough to tell us whether all of them are. But yes, definitely a temporary hack.  I just didn't want to add 20 extra boilerplate functions covering every possible table request.

Reply:
I hear that for sure. And I definitely don't think adding lots of extra functions would be necessary. But using a `table` argument into this method wouldn't be a big change, but would make this check more clear.

Reply:
Are you asking me to only have it extract the particular table that is requested, rather than doing all of them whenever anything is requested? Or are you just wanting to line up the check for existence with the table that's being requested, and then generate all of the tables? To have the check for existence work for any table, I would need to add all of the EIA861 tables to the `__init__()` function (setting them all to None initially), or I would need to do a check like `if table not in self._dfs.keys()` but that would also come out true even if someone requested a gobbledygook table name that shouldn't exist.

Reply:
I guess I was trying to make the call to the `etl_eia861` function clearly not table specific, since with the current setup, it's much more efficient to do them all at once, and putting the table name into the call signature and then still doing all of the tables feels misleading somehow.

Reply:
It is not a big deal, especially because this is temporary.. right? But it feels bad to have the only one table being checked here in this if statement. Can you check if any of the  `self._dfs` that end in `eia861` are `None`?

Reply:
Not unless I go to the trouble of setting them all to None when I initialize the object -- take a look at the `__init__()` method if you haven't already. Otherwise they simply don't exist until the ETL is run.  And yes, all of this will go away when 861 is part of the DB, and behaves like everything else normally.